io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -71,17 +76,18 @@ private:
    TBufferMerger(const TBufferMerger &);
 
    /** TBufferMerger has no copy operator */
-   TBufferMerger& operator=(const TBufferMerger&);
+   TBufferMerger &operator=(const TBufferMerger &);
 
    void Push(TBufferFile *buffer);
-   void Listen();
+   void WriteOutputFile();
 
-   std::mutex fFilesMutex;                                       //< Mutex used to lock fAttachedFiles
+   const std::string fName;
+   const std::string fOption;
+   const Int_t fCompress;
```

Comment:
As opposed to not being `const`. What's the meaning of const int/string? The compiler can deduce this based on the fact that those are allocated on the stack so if it is there for optimization I think it is redundant. If you are telling the reader that you only touch this in the constructors, could be good to know. I don't think this is a common practice in ROOT, however.